config/default/kustomization.yaml

```diff
@@ -73,3 +73,46 @@ patchesStrategicMerge:
 #     kind: Service
 #     version: v1
 #     name: webhook-service
+
+# If you change `metadata.name` in `config/components/secret.yaml` and `internalCertManagement.secretName` in `config/components/manager/controller_manager_config.yaml`,
```

Comment:
Just remove these instructions. It's not a primary use case.

Reply:
Sure.

Reply:
Done.